src/operator/controllers/external_traffic/network_policy.go

```diff
@@ -54,7 +56,14 @@ func NewNetworkPolicyHandler(
 	ingressControllerIdentities []serviceidentity.ServiceIdentity,
 	ingressControllerALBAllowAll bool,
 ) *NetworkPolicyHandler {
-	return &NetworkPolicyHandler{client: client, scheme: scheme, allowExternalTraffic: allowExternalTraffic, ingressControllerIdentities: ingressControllerIdentities, ingressControllerALBAllowAll: ingressControllerALBAllowAll}
+	return &NetworkPolicyHandler{
+		client:                       client,
+		scheme:                       scheme,
+		allowExternalTraffic:         allowExternalTraffic,
+		ingressControllerIdentities:  ingressControllerIdentities,
+		ingressControllerALBAllowAll: ingressControllerALBAllowAll,
+		enforcementConfig:            enforcement.GetConfig(),
```

Comment:
since you're already querying the enforcementConfig here, no point in passing in the allowExternalTraffic value as argument (it's part of the enforcement config). Remove the input argument and just use enforcementConfig all around.

Reply:
Actually, I don't use `enforcementConfig` I meant to do it but eventually kept the same signature. I will remove this line

Reply:
IMO better use the enfocementConfig here than to input the specific flag(s), but it's a style thing, so you choose.